### PR TITLE
[session01] #314 조대흠 앙상블 과제 & 퍼셉트론 딥러닝 기초 사전과제 제출합니다.

### PR DESCRIPTION
**Pull request by damisoda**
### 🔍 어려웠던 점 및 새로 알게 된 점 간단히 정리
앙상블에 대해서 처음 알게 되었고, 이렇게 한개 이상의 모델들에 대해서 예측값을 비교하여 최종 결과를 도출하는 방식이 단일 모델보다 정확할수있다는 점이 신기했다.

### 🔍 질문 및 하고 싶은 말
직접 튜닝을 할 수 있는 마지막 문제가 굉장히 유익했던것 같습니다. 감사합니다!

---
### ✅ PR Checklist
PR이 다음 요구 사항을 충족하는지 확인하세요.
- [X] 본인 브랜치 → develop 브랜치 로 PR 합니다.

### CODE REVIEW

**Comment by jiwon-080** [comment]
랜덤포레스트, XGBoost, LightGBM을 이용한 소프트 보팅 + (각각의 모델 파라미터는 튜닝없이 따로 결정한 것인가요?) 결과 Accuracy 0.80 으로 아주아주 좋게 나왔습니다! (참고로 XGB 단독 + 그리드서치로는 0.75정도가 나옵니다. 보팅을 하니 더 좋은 정확도가 나왔네요)

사전과제도 잘 읽어보았습니다. 엘모 스터디 기초에서 했던 내용과 거의 비슷한 것 같죠?? 모든 핵심적인 내용을 잘 정리해주신 것 같아요!
과제 수고 많으셨습니다 👍👍👍
